### ISSUE TITLE
Safari Problem - Starting Crop Zoom is Too Small 

### ISSUE THREAD

**Issue by bensenescu** [nanxiaobei/antd-img-crop#107]
Hi sorry, this is my first issue I've written so apologies if its poorly formatted.

I've been developing an application using ant design using chrome as my main browser. I noticed today that in Safari that the crop zoom starts out extremely small in safari and I can't seem to find a good solution. 

Below I'll attach two images of what it looks like in each browser:

**Chrome**
<img width="652" alt="Screen Shot 2020-12-21 at 11 12 55 PM" src="https://user-images.githubusercontent.com/44480372/102848299-82bc9900-43e2-11eb-8669-fb188d7b40ee.png">

**Safari**
<img width="520" alt="Screen Shot 2020-12-21 at 11 13 02 PM" src="https://user-images.githubusercontent.com/44480372/102848294-7c2e2180-43e2-11eb-9c53-f536cdbf7b2a.png">

Here is my component for cropping with the dragger component from the antd library:

```
import React from 'react';
import ImgCrop from 'antd-img-crop';
import { Upload } from 'antd';

export interface CropDraggerProps {
  aspectRatio: number;
  grid?: boolean;
  beforeUpload: (file: File) => boolean;
  children: React.ReactNode;
}

const { Dragger } = Upload;

const CropDragger: React.FC<CropDraggerProps> = ({
  aspectRatio, grid, beforeUpload, children,
}: CropDraggerProps) => (
  <ImgCrop
    aspect={aspectRatio}
    grid={grid}
    modalOk="Save"
  >
    <Dragger
      name="picture"
      beforeUpload={beforeUpload}
      accept="image/*"
    >
      {children}
    </Dragger>
  </ImgCrop>
);

export default CropDragger;
```

Let me know if you need any other information or if this has already been addressed somewhere.

**Comment by nanxiaobei** [comment]
This should be fixed in `3.13.0`.

https://github.com/nanxiaobei/antd-img-crop/issues/82
https://github.com/nanxiaobei/antd-img-crop/pull/85

**Comment by bensenescu** [comment]
This fixed it, Thank you!
